.cirrus.yml

```diff
@@ -292,12 +292,14 @@ testing_task:
     setup_environment_script: '$SCRIPT_BASE/setup_environment.sh |& ${TIMESTAMP}'
     unit_test_script: '$SCRIPT_BASE/unit_test.sh |& ${TIMESTAMP}'
     integration_test_script: '$SCRIPT_BASE/integration_test.sh |& ${TIMESTAMP}'
+    ginkgo_node_logs_script: 'cat $SCRIPT_BASE/test/e2e/ginkgo-node-*.log || echo "Ginkgo node logs not found"'
```

Comment:
Oops, that's a bad path: should be `$CIRRUS_WORKING_DIR/test/e2e/ginkgo-node-*.log`